java/mtsj/core/src/main/resources/application.properties

```diff
@@ -52,9 +52,14 @@ mythaistar.hourslimitcancellation=1
 
 ##Mailing properties
 #enabled set to false will use the MailMock service
-spring.mail.enabled=false
-spring.mail.host=smtp.gmail.com
-spring.mail.username=mythaistarrestaurant@gmail.com
-spring.mail.password=mythaistarrestaurant2501
-spring.mail.properties.mail.smtp.auth=true
-spring.mail.properties.mail.smtp.starttls.enable=true
+spring.mail.enabled=true
+#spring.mail.host=smtp.gmail.com
+spring.mail.host=de-nagios01.corp.capgemini.com
```

Comment:
environment config should only go to `config/application.properties`. Please read the header of these properties files and guide:
https://github.com/devonfw/devon4j/blob/develop/documentation/guide-configuration.asciidoc#externalized-configuration